shared/src/api/BasicMultipleChoiceQuestion.ts

```diff
@@ -121,8 +121,7 @@ export function basicMultipleChoiceMetaGenerator(
 
     random.shuffle(answers)
 
-    const markdown =
-      questions[i].frame === undefined ? ownt["text"] : questions[i].frame!(l, ownt["text"])
+    const markdown = questions[i].frame!(l, ownt["text"])
```

Comment:
Is change breaks the functionality.

The linter complains about the `!` at the end. This is an implicit assertion that `.frame` is not undefined there. But we already checked for that case, which makes the assertion unnecessary. 

So the correct change would be to just remove `!` resulting in 
```
questions[i].frame === undefined ? ownt["text"] : questions[i].frame(l, ownt["text"])
```

Reply:
The ! is necessary in typescript < 5.5, only >= 5.5 is able to infer this correctly.

Reply:
Ah, this explains it. The PR includes an update from typescript 5.4 to 5.5

Reply:
I changed it back and will add the ! if I go back to < 5.5